docs/platforms/powershell/data-management/data-collected.mdx

```diff
@@ -0,0 +1,65 @@
+description: "See what data is collected by the Sentry SDK."
+sidebar_order: 1
+---
+
+Sentry takes data privacy very seriously and has default settings in place that prioritize data safety, especially when it comes to personally identifiable information (PII) data. When you add the Sentry SDK to your application, you allow it to collect data and send it to Sentry during the runtime of your application.
+
+The category types and amount of data collected vary, depending on the integrations you've enabled in the Sentry SDK. Here's a list of data categories the Sentry .NET SDK collects:
+
+## HTTP Headers
+
+By default, the Sentry SDK sends HTTP headers to Sentry but redacts any headers that may contain sensitive data. (See the [list of headers](https://github.com/getsentry/sentry-dotnet/blob/main/src/Sentry/Internal/RedactedHeaders.cs#L7) that are redacted).
+
+## Cookies
+
+By default, the Sentry SDK doesn't send cookies.
+
+If you want to send cookies, set the option `SendDefaultPii = true` when initializing the Sentry SDK.
+
+A [data scrubber](/platforms/dotnet/data-management/sensitive-data/) can be used to filter information before sending data to Sentry.
+
+## Information About Logged-in User
+
+By default, the Sentry SDK doesn't send any information about the logged-in user, such as email address, user ID, or username. Even if enabled, the type of logged-in user information you'll be able to send depends on the integrations you enable in Sentry's SDK. Most integrations won't send any user information. Some will only set the user ID, but there are a few that will set the user ID, username, and email address.
+
+To start sending logged-in user information, set the option `SendDefaultPii = true` when initializing the Sentry SDK.
+
+## Users' IP Address
+
+By default, the Sentry SDK doesn't send the user's IP address. Even if enabled, whether you're able to send the user's IP address or not, will depend on the integrations you enable in Sentry's SDK. Most integrations won't set the user's IP address at all.
+
+To enable sending the user's IP address, set the option `SendDefaultPii = true` when initializing the Sentry SDK.
+
+## Request URL
+
+By default, the [HttpRequestUrl](https://learn.microsoft.com/en-us/dotnet/api/System.UriComponents) of outgoing and incoming HTTP requests is **always sent to Sentry**. This includes the Scheme, Host, Port, LocalPath, and Query data.
+
+If `SendDefaultPii` is set to `true` then the [AbsoluteUri](https://learn.microsoft.com/en-us/dotnet/api/System.UriComponents) of outgoing and incoming HTTP requests will be sent to Sentry (including the Scheme, UserInfo, Host, Port, LocalPath, Query, and Fragment data).
+
+Depending on your application, this could contain PII data.
+
+## Request Query String
+
+The full request query string of outgoing and incoming HTTP requests is **always sent to Sentry**. Depending on your application, this could contain PII data.
+
+## Local Variables In Stack Trace
```

Comment:
PowerShell might have local variables actually, but I don't know.
@vaind does it?

Reply:
I've updated the powershell docs so these basically say "see the data collected by the .net skd" now. If there are some minor differences, we can always tweak that to say "except A, B and C as detailed below".

Reply:
> PowerShell might have local variables actually, but I don't know.
> @vaind does it?

Yes but we don't capture them at the moment